src/components/templates/Playa/MyAvatar.tsx

```diff
@@ -77,7 +77,7 @@ const MyAvatar: React.ForwardRefRenderFunction<HTMLDivElement, PropsType> = (
     stateInitialized.current = true;
   }, [serverSentState, setMyLocation, setBikeMode, setVideoState]);
 
-  const arrowMoveIntervalRef = useRef<NodeJS.Timeout | null>(null);
+  const arrowMoveIntervalRef = useRef<number | null>(null);
```

Comment:
This type error has been lurking for a while, not sure why it wasn't caught earlier but the compiler spontaneously started giving errors about it that wouldn't go away. Easy fix, have to include here due to build failures otherwise.

Reply:
Is the value you changed it to a valid value for how we use`arrowMoveIntervalRef`? (I assume yes, but i'd rather ask and get an 'all good' than not ask and find out it was an issue.